test/test.js

```diff
@@ -76,6 +76,19 @@ describe('cacheman-file', function() {
     });
   });
 
+  it('should sanitize key', function(done) {
```

Comment:
I do agree that the developer is agnostic to what's going on behind the scenes, but the tests should know. I like the updated test description as well. I think you can check the cached key against the sanitized key. If those match up, then we know that the key is properly being sanitized? How does that sound?

My plan is to update the tests to show how much coverage we have for the code.